sdk/pinocchio/src/account_info.rs

```diff
@@ -379,6 +378,75 @@ impl AccountInfo {
         Ok(())
     }
 
+    /// Zero out the the account's data_len, lamports and owner fields, effectively
```

Comment:
This method probably needs to be marked `unsafe` since it does not check whether the account is already borrowed or not. We should add a comment along the lines:
```rust
/// # Safety
///
/// This method is unsafe because it does not check if the account data is already
/// borrowed. It should only be called when the account is not being used.
```